plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/BaseClickHouseConnectorSmokeTest.java

```diff
@@ -53,4 +55,46 @@ public void testShowCreateTable()
                         "   engine = 'LOG'\n" +
                         ")");
     }
+
+    @Override
+    @Test
+    public void testDeleteAllDataFromTable()
+    {
+        // Override because default storage engine doesn't support delete
+        String tableDefinition = "(" +
+                " regionkey BIGINT NOT NULL," +
+                " name VARCHAR(25) NOT NULL," +
+                " comment VARCHAR(115) NOT NULL" +
+                "    ) WITH (engine='MergeTree',  order_by=ARRAY['regionkey'])";
```

Comment:
Is it possible to use CTAS statement instead?

Reply:
can you please give an example how @ebyhr ? (while being able to specify the table properties)

the issue i am running into is like this
1. the tables can be created but the column will all nullable. 
2. clickhouse requires atleast one non null column for the order_by

PS: 2 can be overcome by a clickhouse property that allows nullable order_by keys, however its not recognized in trino yet https://clickhouse.com/docs/en/operations/settings/settings#allow-nullable-key